dash-pipeline/SAI/templates/dashsai.cpp.j2

```diff
@@ -455,6 +455,14 @@ sai_status_t DashSai::getPortAttribute(
 
                 break;
 
+            case SAI_PORT_ATTR_OPER_STATUS:
+
+                attr->value.s32 = SAI_PORT_OPER_STATUS_UP;
```

Comment:
this should speed up ptf test execution in github pipeline from 6 minutes to about 5 seconds if everything is right :)

Reply:
actually it's 2 seconds now XD, entire build and tests was 30min at the beginning, now all is about 6minutes total, huge improvement

Reply:
You're a star!

Reply:
@kcudnik I cloned your branch and built evertying. Running `make run-all-tests` still takes several minutes, and I see a lot of the following below. Did I miss something?
```
Get default vlan...
For Common platform, Only check Port status.
Turn up port 281474976710657
port 0:281474976710657 is not up, status: 0. Retry. Waiting for Admin State Up.
port 0:281474976710657 is not up, status: 0. Retry. Waiting for Admin State Up.
port 0:281474976710657 is not up, status: 0. Retry. Waiting for Admin State Up.
port 0:281474976710657 is not up, status: 0. Retry. Waiting for Admin State Up.
port 0:281474976710657 is not up, status: 0. Retry. Waiting for Admin State Up.
port 0:281474976710657 is not up, status: 0. Retry. Waiting for Admin State Up.
port 0:281474976710657 is not up, status: 0. Retry. Waiting for Admin State Up.
port 0:281474976710657 is not up, status: 0. Retry. Waiting for Admin State Up.
```

Reply:
I've confirmed this branch build in under 3m and tests run in 1m12s, a huge improvement. Great job!